cypress/e2e/with-users/domains/list.spec.ts

```diff
@@ -7,6 +7,6 @@ context("DNS", () => {
   });
 
   it("renders the correct heading", () => {
-    cy.get("[data-testid='section-header-title']").contains("DNS");
+    cy.get("[data-testid='main-toolbar-heading']").contains("DNS");
```

Comment:
Depending on how many failures are related to this, it might make more sense to rename the testid in toolbar itself instead. There are more places where we use this pattern.

Reply:
I think it's only 4 other failures in Cypress.